CONTRIBUTING.md

```diff
@@ -45,45 +45,75 @@ If you would like to contribute, please follow these steps:
     ```
     git checkout -b <dev-your-bugfix-or-feature>
     ```
-4. [pre-commit](https://pre-commit.com/) is used to run auto-formatting and linting tools. Please follow the steps in the link provided to install `pre-commit` locally.  `pre-commit` can be installed as a git hook to verify your staged changes adhere to the project's style and format requirements (configuration defined in [pyproject.toml](/pyproject.toml)).
-5. After you've installed `pre-commit` via `pip` , `homebrew`, or `conda`, check the version to verify installation:
-    ```
-    $ pre-commit --version
-    ```
-
-6. Initialize the pre-commit hooks included within the root directory of `inundation-mapping`:
-
+4. Pre-commit linting system:
+
+   [pre-commit](https://pre-commit.com/) is used to run auto-formatting and enforce styling rules.
+   It is a critical part of development and is enforced during git check-ins. However, you also need some tools need to be installed and configured on your machine to pre check the linting rules. Key tools are automatically included **inside the docker container** if you want to do linting tests there.
+
+   The next steps are only if you want to do linting tests **outside the docker container**. If so, you need to follow the notes below. If not, skip directly to Step 5 (pre-commit configuration).
```

Comment:
This is not the case the way things are currently configured. `pre-commit` needs to be installed at the system level. I'd recommend removing this line altogether.

Reply:
I am not sure what you mean. Would you mind editing and checking it in as you see fit?

Reply:
Yea I'll delete line 53 and commit the change, but for your reference, we need to have the `pre-commit` package installed on each individual machine, not only available within the docker container.  In order to run `pre-commit install`, which runs the pre-commit git hooks, the package needs to be installed outside of the container. 
 
The verbiage here "The next steps are only if you want to do linting tests" makes it seem as though it is optional, which is not the case, given the fact that now the pre-commit hooks are part of our CI system, and need to pass before a PR is merged. It is easiest to catch things at the commit level than the PR level.